catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -837,4 +837,11 @@ private void addFileToMap(Map<String, List<File>> fileMap, File file) {
       throw new IllegalArgumentException("Invalid transformer transformerId: " + transformerId, e);
     }
   }
+
+  private File getDefaultIngestLocation() {
+    String home =
+        AccessController.doPrivileged(
```

Comment:
❓ Was this necessary? It should have already had privileges to read this property.

Reply:
It does ensure that it doesn't bleed through to clients of this API